experimental/client-csharp/src/Wandb.Sdk/WandbClient.cs

```diff
@@ -0,0 +1,387 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Collections.Generic;
+using System.Buffers.Binary;
+using Newtonsoft.Json;
```

Comment:
I believe JSON conversion is built into .NET now: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-9-0